child_compassion/data/gmc_action.xml

```diff
@@ -41,14 +41,14 @@
             <field name="auto_process">1</field>
             <field name="success_method">details_answer</field>
         </record>
-        <!-- Get ICP details -->
+        <!-- Get FCP details -->
         <record id="icp_details" model="gmc.action">
             <field name="direction">out</field>
-            <field name="name">ICPDetails</field>
+            <field name="name">FCPDetails</field>
             <field name="model">compassion.project</field>
-            <field name="connect_answer_wrapper">ICPResponseList</field>
+            <field name="connect_answer_wrapper">FCPResponseList</field>
```

Comment:
Sorry, but this should not be changed otherwise our messages with US won't work.